src/components/FiltersBar/FiltersBar.styles.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/shared/components/Text'
+import { SvgGlyphHide } from '@/shared/icons'
+import { colors, sizes, transitions } from '@/shared/theme'
+
+export const FilterContentContainer = styled.div`
+  display: grid;
+  gap: ${sizes(3)};
+`
+
+export const MobileFilterContainer = styled.div`
+  display: grid;
+  padding-bottom: ${sizes(6)};
+  gap: ${sizes(3)};
+`
+
+export const FiltersContainer = styled.div<{ open: boolean }>`
+  display: ${({ open }) => (open ? 'flex' : 'none')};
+  justify-content: space-between;
+  padding: ${sizes(4)};
+  background-color: ${colors.gray[900]};
+  will-change: opacity;
+  transition: opacity ${transitions.timings.sharp} ${transitions.easing};
+
+  &.filters-active,
+  &.filters-exit {
+    opacity: 1;
+  }
+
+  &.filters-exit-active,
+  &.filters-enter {
+    opacity: 0;
+  }
+`
+
+export const FiltersInnerContainer = styled.div`
+  display: grid;
+  gap: ${sizes(4)};
+  grid-auto-flow: column;
+  grid-auto-columns: max-content;
+`
+
+export const OtherFilterStyledText = styled(Text)`
+  margin-bottom: ${sizes(2)};
+  display: flex;
+  align-items: center;
+`
+
+export const OtherFilterStyledIcon = styled(SvgGlyphHide)`
+  margin-right: ${sizes(2)};
+
+  & path {
+    fill: currentColor;
+    stroke: currentColor;
+  }
```

Comment:
I think those style cause the icon to not look as it should - compare with the design

Reply:
Seems pretty clear to me:
<img width="279" alt="Screenshot 2021-10-11 at 09 41 27" src="https://user-images.githubusercontent.com/12646744/136751312-cc0fa1e2-9e41-4392-b452-58e70939334d.png">
<img width="279" alt="Screenshot 2021-10-11 at 09 41 07" src="https://user-images.githubusercontent.com/12646744/136751317-0a3e4595-879d-4844-96aa-9a4ea94a31d9.png">